lib/docker_registry2.rb

```diff
@@ -11,8 +11,8 @@ def self.connect(uri = 'https://registry.hub.docker.com', opts = {})
     @reg = DockerRegistry2::Registry.new(uri, opts)
   end
 
-  def self.search(query = '')
-    @reg.search(query)
+  def self.search(query = '', records = 100)
```

Comment:
Is it a bit strange that here the params are `query` and `records`, but in `reg.search()` they are `query` and `record_count`? Should we not be consistent?

Reply:
ok, i will align them :+1: